librocksdb_sys/build.rs

```diff
@@ -16,7 +16,7 @@ macro_rules! t {
 fn main() {
     let mut cfg = build_rocksdb();
 
-    cfg.cpp(true).file("crocksdb/c.cc");
+    cfg.cpp(true).flag_if_supported("-fno-rtti").file("crocksdb/c.cc");
```

Comment:
why do we need to check no-rtti here?

Reply:
RocksDB abandon `dynamic_cast<T>`, we need to make consistency with RocksDB.